### ISSUE TITLE
Triage Faker import Error

### ISSUE THREAD

**Issue by glassresistor** [codeforamerica/intake#1211]
We currently use Faker to generate data for our tests and to load initial data.

For some reason we are getting an import error in Lambda.  Since it would be problematic to database queries from the build server it would be good to figure out why and fix it.

Things to try
- Rebuild local virtualenv
- Upgrade to the new faker and factory_boy packages.
- Install from git and see if this is related https://github.com/joke2k/faker/issues/332
- Fork and fix
- Dump fixtures and load from s3 bucket.

**Comment by bengolder** [comment]
@glassresistor if you link a stack trace or more info on steps to reproduce, I could help debug this

**Comment by glassresistor** [comment]
@bgolder the ticket linked here joke2k/faker#332 has the same stack trace.  

If you look on the zappa branch I've updated faker and factory_boy to the newest versions and am not installing faker-factory anymore.

You're well come to take a look but it requires being able to deploy to zappa to debug it.

I've pulled it down locally and added some print statements to see what code path and variables were.  So far its not fixed the issue.

The options currently left are:
- Fork and fix
- Dump fixtures and load from s3 bucket.
